src/bpftrace.cpp

```diff
@@ -2196,6 +2197,11 @@ std::string BPFtrace::resolve_usym(uintptr_t addr,
         psyms = pid_sym_[pid];
       }
     }
+    else
+    {
+      // no user symbol caching, create new bcc cache
+      psyms = bcc_symcache_new(pid, &symopts);
```

Comment:
The old code had a matching `bcc_free_symcache` when caching was disabled - I assume we'll leak here without it